BezierSwift/Sources/Components/Badges/ChatBadge.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  ChatBadge.swift
+//
+//
+//  Created by 구본욱 on 8/9/24.
+//
+
+import SwiftUI
+
+private enum Constant {
+  static let backgroundColor: BezierColor = .bgWhiteHighest
+  static let iconColor: BezierColor = .fgBlackDarker
+}
+
+/// ChatBadge는 Avatar와 함께 사용되며 아이콘을 사용해 Root(Avatar)의 상태가 Private임을 알려줍니다.
+struct ChatBadge: View {
+  private let configuration: ChatBadgeConfiguration
+  
+  /// 주어진 ChatBadgeConfiguration에 따른 ChatBadge를 생성합니다.
+  /// - Parameter configuration: ChatBadge의 사이즈를 결정짓는 configuration 객체입니다.
+  public init(configuration: ChatBadgeConfiguration) {
+    self.configuration = configuration
+  }
+
+  public var body: some View {
+    ZStack(alignment: .center) {
+      BezierIcon
+        .lock
+        .image
```

Comment:
ViewModifier와 구분지어주기 위해서 image까지는 한줄로 써주는 게 좋을 것 같아요
```suggestion
      BezierIcon.lock.image
```